installer.sh

```diff
@@ -44,7 +44,12 @@ function install_service {
     NAME=${CATALOG_RECORD[0]//,}
     PLAN=${CATALOG_RECORD[1]//,}
     echo $NAME $PLAN
-    RETURN=($(cf create-service ${NAME} ${PLAN} "my$1"))
+	if [ -z "$2" ];
```

Comment:
Sine on gotapaas.eu new service with a name kb-postgres-something was introduced, no, grep does not returns postgresql93.